File: Homeworks/C# Part 2/06.StringsAndTextProcessing/04.SubStringInText/SubStringInText.cs
using System;

class SubStringInText
{
    static void Main()
    {
        string pattern = Console.ReadLine().ToLower();
        string text = Console.ReadLine().ToLower();
        int count = 0;
        int startIndex = 0;
        int foundIndex = 0;
        while (true)
        {
            foundIndex = text.IndexOf(pattern, startIndex);
            if (foundIndex < 0)
            {
                break;
            }
            count++;
            startIndex = foundIndex + 1;
        }
        Console.WriteLine(count);
    }
}

File: Homeworks/JS Fundamentals/11. Strings/03. Sub-string in text/sub-string-in-text.js
function solve(args) {
	var pattern = args[0].toLowerCase(),
	    text = args[1].toLowerCase(),
        count = 0,
        startIndex = 0,
        foundIndex = 0;
    while (true)
    {
        foundIndex = text.indexOf(pattern, startIndex);
        if (foundIndex < 0)
        {
            break;
        }
        count += 1;
        startIndex = foundIndex + 1;
    }
    console.log(count);
}